packages/data/tests/unit/models/navi-dimension-test.ts

```diff
@@ -0,0 +1,40 @@
+import { module, test } from 'qunit';
+import { setupTest } from 'ember-qunit';
+import NaviDimensionModel from 'navi-data/models/navi-dimension';
+import { TestContext as Context } from 'ember-test-helpers';
+//@ts-ignore
+import { setupMirage } from 'ember-cli-mirage/test-support';
+import NaviMetadataService from 'navi-data/services/navi-metadata';
+import { DimensionColumn } from 'navi-data/adapters/dimensions/interface';
+import DimensionMetadataModel from 'navi-data/models/metadata/dimension';
+
+interface TestContext extends Context {
+  metadataService: NaviMetadataService;
+}
+
+module('Unit | Model | navi dimension', function(hooks) {
+  setupTest(hooks);
+  setupMirage(hooks);
+  hooks.beforeEach(async function(this: TestContext) {
+    this.metadataService = this.owner.lookup('service:navi-metadata') as NaviMetadataService;
```

Comment:
Can we just assign this as a top level object instead of having to set `this: TestContext` in the tests

Reply:
I know we have done that in the past. Instead of relying on global like variables between tests, I feel this better documents what is generally available in the test environment.

Reply:
If we didn't have to specify `this` for every test I wouldn't have any objections. I don't feel too strongly either way, but I'm more in favor of how we did it before